gulpfile.js

```diff
@@ -84,7 +84,7 @@ gulp.task('build:js', function() {
         //.pipe(sort())
         .pipe(concat('app.js'))
         .pipe(gulp.dest((BUILD_DEST + 'js')))
-        .pipe(uglify())
+        //.pipe(uglify())
```

Comment:
Is this intentionally commented out? If so the following two lines don't make much sense either